packages/logs/src/test/stubCspEventListener.ts

```diff
@@ -0,0 +1,31 @@
+export function stubCspEventListener() {
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const originalAddEventListener = document.addEventListener
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏you could use `spyOn(document, 'addEventListener').and.callFake(...)`